<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LitteLiders</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="terminalInfoButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>단말기 관리</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="courseInfoButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>운행 가능한 코스 보기</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>위도</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="latitudeText">
         <property name="text">
          <string>latitude</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>경도</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="longitudeText">
         <property name="text">
          <string>longitude</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>속도</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="speedText">
         <property name="text">
          <string>speed</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>나가기</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>381</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mapLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>terminalInfoButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>terminalInfoButtonEvent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>courseInfoButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>courseInfoButtonEvent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>terminalInfoButtonEvent()</slot>
  <slot>courseInfoButtonEvent()</slot>
 </slots>
</ui>
